core/controllers/library_test.py

```diff
@@ -23,12 +23,12 @@
 from core.domain import rating_services
 from core.domain import rights_manager
 from core.domain import user_services
-from core.platform.taskqueue import gae_taskqueue_services as taskqueue_services
+from core.platform.taskqueue import \
```

Comment:
ditto, drop \

If isort is doing this automatically we need to find a way to make it not do that.

Ditto elsewhere -- this occurs other times too.

Reply:
Okay, got it! I will take care of all such occurrences.